mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextEditorImpl.java

```diff
@@ -190,6 +191,9 @@ public void mouseWheelMoved(MouseWheelEvent e) {
                 }
             }
         });
+        // Setting TEXT_CURSOR for text area regardless if it is editable or readonly - for both it makes sense.
+        // TODO still not ideal - initially most often it opens with DEFAULT_CURSOR, moving to line numbers panel and back to editor it shows TEXT_CURSOR properly
```

Comment:
it works well on Linux (Fedora), I'll check on my macbook

Reply:
@pskowronek just a quick update on this - it reproduces on my macbook but couldn't pinpoint the problem yet

Reply:
ok, this also happens with a simple JFrame and JTextArea on macOS so it doesn't look like a result of our code - maybe it would be fixed with a newer version of Java
but I don't see a reason to block this PR on that